.github/workflows/aws_deploy.yml

```diff
@@ -129,24 +129,46 @@ jobs:
           succeeded=$(aws ecs describe-tasks --cluster $cluster --tasks $id --query "tasks[0].stopCode == 'EssentialContainerExited' && to_string(tasks[0].containers[0].exitCode) == '0'")
           if [ $succeeded == true ]; then exit 0; else exit 1; fi
 
-      - name: Download service task definition
+      - name: Download app service task definition
         env:
           app_task_definition: ${{ inputs.aws_resource_prefix }}-app
         run: |
           aws ecs describe-task-definition --task-definition $app_task_definition --query taskDefinition > app-task-definition.json
 
-      - name: Update image ID
+      - name: Download sidekiq service task definition
```

Comment:
I think this pipeline would make more sense conceptually if it did all the app stuff and then all the sidekiq stuff -and neither of these steps take long enough to need to do all the prep then all the deploy.

Reply:
Agree! I'll update the order 👍